packages/testsuite/cypress/e2e/transaction/transactions-configuration-subsystem-transaction-jdbc.cy.ts

```diff
@@ -20,20 +20,20 @@ describe("TESTS: Configuration => Sybsystem => Transaction => Path", () => {
 
   const datasourceBase = {
     connectionUrl:
-      "jdbc:h2:mem:test;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE;MODE=${wildfly.h2.compatibility.mode:REGULAR}",
+      "jdbc:h2:mem:test;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE;MODE=\\${wildfly.h2.compatibility.mode:REGULAR}",
     driverName: "h2",
   };
 
   const nonEmptyDS = {
     ...datasourceBase,
     name: "nonEmptyDS",
-    jndiName: "java:jboss/datasources/nonEmptyDS,",
+    jndiName: "java:jboss/datasources/nonEmptyDS",
   };
 
   const nonEmptyDSUpdated = {
     ...datasourceBase,
     name: "nonEmptyDS",
```

Comment:
I think there should be a different name, e.g. nonEmptyDS -> nonEmptyDSUpdated. There will be no duplication in the `before` method. And the related test will start make sense :)

Reply:
Thanks, this help to avoid to a duplicate resource exception.